tsconfig.json

```diff
@@ -5,7 +5,8 @@
     "outDir": "lib",
     "esModuleInterop": true,
     "declaration": true,
-    "strict": true
+    "strict": true,
+    "typeRoots": ["./node_modules/@types"]
```

Comment:
According to this https://www.typescriptlang.org/tsconfig#typeRoots we don't need `typeRoots`, it is already used by default